src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -496,6 +502,11 @@ private String postRequest(PostMethod httppost) throws UnsupportedEncodingExcept
 
     }
 
+    private <T> String serialize(T o) throws IOException {
```

Comment:
Was actually reusing this [BitbuckerServerAPIClient.serialize](https://github.com/jenkinsci/bitbucket-branch-source-plugin/blob/a02519c7cfbde64cd309ccb8e7b08ebbdf0efed9/src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java#L443) and, as pointed out, short of completely refactoring the two clients into some type of shared abstract client or the like.